pkg/cmd/api/api.go

```diff
@@ -167,6 +174,23 @@ func NewCmdApi(f *cmdutil.Factory, runF func(*ApiOptions) error) *cobra.Command
 			    }
 			  }
 			'
+
+			# get the percentage of forks for the current user
+			# without --slurp you will get a different percentage for each page
```

Comment:
I can try to be more clear - or correct, at least - but honestly I feel this blurb is convoluted anyway. My original intention was to pass the output to PowerShell's `ConvertFrom-Json` since all our CI scripts are PowerShell: works cross-platform and is installed on all build agents already, unlike `jq`. But PowerShell is not used in any other examples in the CLI that I've seen, so I was just trying to think of something that would set `--slurp` apart for `jq` which never really had a major problem dealing with the otherwise invalid JSON - multiple pages of arrays or objects which, as a whole, is invalid JSON.

If you have a better example in mind, I'm all ears. Or if you don't mind me mentioning PowerShell, I can do that instead and it would be much clearer.

Reply:
I think I'd be happy just to remove the `# without --slurp` line.